src/sage/groups/abelian_gps/abelian_group_morphism.py

```diff
@@ -25,6 +25,8 @@
 
 
 def is_AbelianGroupMorphism(f):
+    from sage.misc.superseded import deprecation
+    deprecation(38103, "The function is_AbelianGroupMorphism is deprecated; use 'isinstance(..., AbelianGroupMorphism)' instead.")
```

Comment:
Isn't this line too long? Why are the linters not triggering 🤔

Reply:
I've broken some of the long lines in 8619affb424